exercises/isbn-verifier/.meta/src/reference/java/IsbnVerifier.java

```diff
@@ -0,0 +1,39 @@
+class IsbnVerifier {
+
+    private String isbn;
+
+    boolean isValid(String s) {
+
+        this.isbn = s.replace("-", "");
+        int total = 0;
+
+        if (isbn.length() != 10) {
+            return false;
+        } else {
+            for (int i = 0; i < isbn.length() - 1; i++) {
+                if(Character.isDigit(isbn.charAt(i))) {
```

Comment:
I would extract `isbn.charAt(i)` into a local variable since you're using it twice :)